---
title: "Integrative genomic analysis of mouse and human hepatocellular carcinoma"
collection: publications
permalink: /publications/2018-10-16_genomic_analysis_of_mouse_human_HCC
venue: "PNAS (Proceedings of the National Academy of Sciences of the United States of America)" 
excerpt: 'Hepatocellular carcinoma (HCC) research has been hampered by the absence of consensus mouse models with clearly defined molecular features faithfully recapitulating human HCC. Here we tackle this gap by implementing a cross-species comparative analysis between a large cohort of patients and four diverse mouse models focused on clinically and therapeutically relevant aspects of genomic and transcriptomic profiles and propose two of these models as valid for the study of different stages of human HCC.'
date: 2018-10-16
paperurl: http://michelled3.github.io/files/2018-10-16_genomic_analysis_of_mouse_human_HCC.pdf
---

## Abstract
Cancer genomics has enabled the exhaustive molecular characterization of tumors and exposed hepatocellular carcinoma (HCC) as among the most complex cancers. This complexity is paralleled by dozens of mouse models that generate histologically similar tumors but have not been systematically validated at the molecular level. Accurate models of the molecular pathogenesis of HCC are essential for biomedical progress; therefore we compared genomic and transcriptomic profiles of four separate mouse models [MUP transgenic, TAK1-knockout, carcinogen-driven diethylnitrosamine (DEN), and Stelic Animal Model (STAM)] with those of 987 HCC patients with distinct etiologies. These four models differed substantially in their mutational load, mutational signatures, affected genes and pathways, and transcriptomes. STAM tumors were most molecularly similar to human HCC, with frequent mutations in Ctnnb1, similar pathway alterations, and high transcriptomic similarity to high-grade, proliferative human tumors with poor prognosis. In contrast, TAK1 tumors better reflected the mutational signature of human HCC and were transcriptionally similar to low-grade human tumors. DEN tumors were least similar to human disease and almost universally carried the Braf V637E mutation, which is rarely found in human HCC. Immune analysis revealed that strain-specific MHC-I genotype can influence the molecular makeup of murine tumors. Thus, different mouse models of HCC recapitulate distinct aspects of HCC biology, and their use should be adapted to specific questions based on the molecular features provided here.

[Download paper here](http://michelled3.github.io/files/2018-10-16_genomic_analysis_of_mouse_human_HCC.pdf)
