---
title: "Extracting allelic read counts from 250,000 human sequencing runs in Sequence Read Archive"
collection: publications
permalink: /publications/2019-01-03_extract_250k_SRA_runs
venue: "Pacific Symposium on Biocomputing"
excerpt: " This study provides a rich database of known human variants across SRA samples that can support future meta-analyses of human sequence variation."
date: 2019-01-03
paperurl: http://michelled3.github.io/files/2019-01-03_extract_250k_SRA_runs.pdf
---

## Abstract
The Sequence Read Archive (SRA) contains over one million publicly available sequencing runs from various studies using a variety of sequencing library strategies. These data inherently contain information about underlying genomic sequence variants which we exploit to extract allelic read counts on an unprecedented scale. We reprocessed over 250,000 human sequencing runs (>1000 TB data worth of raw sequence data) into a single unified dataset of allelic read counts for nearly 300,000 variants of biomedical relevance curated by NCBI dbSNP, where germline variants were detected in a median of 912 sequencing runs, and somatic variants were detected in a median of 4,876 sequencing runs, suggesting that this dataset facilitates identification of sequencing runs that harbor variants of interest. Allelic read counts obtained using a targeted alignment were very similar to read counts obtained from whole-genome alignment. Analyzing allelic read count data for matched DNA and RNA samples from tumors, we find that RNA-seq can also recover variants identified by Whole Exome Sequencing (WXS), suggesting that reprocessed allelic read counts can support variant detection across different library strategies in SRA. This study provides a rich database of known human variants across SRA samples that can support future meta-analyses of human sequence variation.

[Download paper here](http://michelled3.github.io/files/2019-01-03_extract_250k_SRA_runs.pdf)
